### PR TITLE
update required ruby version

### PR DESCRIPTION
**Pull request by activemq**
- [x] upgrade ruby version across project to 2.6.0

- [x] add required version using bundler ruby version to enforce version 2.6.0

### CODE REVIEW

File: api_tests/Gemfile.lock [hunk 1]
```diff
@@ -36,5 +36,8 @@ DEPENDENCIES
   pry
   rspec
 
+RUBY VERSION
```
Comment (GuyHarwood) [inline api_tests/Gemfile.lock:39]: did you change this yourself, or auto updated?

Comment (activemq) [reply api_tests/Gemfile.lock:39]: @GuyHarwood this is added automatically as now i have added a ruby version to the Gemfile. Then when we run bundle install, it fails if any other version of ruby is used with the following error message:
"Your Ruby version is 2.4.0, but your Gemfile specified 2.6.0"

**Comment by activemq** [comment]
Please hold off merging as we need to setup ruby version 2.6.0 on the build servers before merging.

File: api_tests/Gemfile.lock (hunk 1 above)
Comment (GuyHarwood) [reply api_tests/Gemfile.lock:39]: Good 👍

**Pull request merged**